urls.py

```diff
@@ -99,7 +99,6 @@
     url(r'^account/', include('corehq.apps.settings.urls')),
     url(r'^project_store(.*)$', rewrite_url),
     url(r'^exchange/', include('corehq.apps.appstore.urls')),
-    url(r'^webforms/', include('touchforms.formplayer.urls')),
```

Comment:
@wpride @snopoke is it true that nothing else besides smsforms was using touchforms? i found this `/webforms/` url was still being exposed. i checked the nginx logs and for the past month it's only had 404 responses returned for bots and such

Reply:
I couldn't find any references to this URL or any of it's sub URLs